WalletWasabi/Blockchain/Keys/KeyManager.cs

```diff
@@ -623,17 +623,45 @@ public Height GetBestHeight()
 			return BlockchainState.Height;
 		}
 	}
+	
+	public int? GetBestTurboSyncHeight()
```

Comment:
Something is off here. This is a getter method that sets a value and save to disk. Finally it return a non-nullable value but the signature of the method says it could be nullable.